content/_data/changelogs/weekly.yml

```
@@ -8630,6 +8630,128 @@
         - daniel-beck
         - wadeck
 
+- version: '2.253'
+  date: 2020-08-18
+  changes:
+  - type: major rfe
+    authors:
+    - oleg-nenashev
+    - slide
+    message: |-
+      Major update of the Alpine-based Jenkins Docker image.
+      Jenkins Docker image for Alpine now uses Alpine 3.12 and AdoptOpenJDK 8u262.
+    references:
+      - url: /doc/upgrade-guide/2.235/#docker-alpine-image-overhaul
+        title: LTS upgrade guide
+  - type: major bug
+    category: regression
+    pull: 4897
+    issue: 63274
+    authors:
+    - Wadeck
+    message: |-
+      Fix button that copies API token to clipboard (regression in 2.238).
+  - type: major bug
```

```suggestion
  - type: bug
```

Also needs to be reordered